### DISCUSSION TITLE
i3 > Sway ~ for gaming and most things

### DISCUSSION THREAD

**Discussion by dillacorn** [i3/i3#6226] [Show and tell]
Distro: Debian Sid
Dots: www.github.com/dillacorn/dotfiles

I just want to thank the devs for writing such an awesome window manager!

It's so easy to use and I love the performance even on lower refresh rate displays for gaming..

Sway supposedly added tearing support but I haven't managed to get it working and the games I do play feel very sluggish at lower refresh rates.. i3 is perfect and feels like there is zero input lag..

Xorg allows me to use VibrantGUI to increase my digital vibrance... Why is this not a thing in wlroots and implemented in Sway config output at the very least is beyond me..

I also love the fact that Xorg window managers such as i3 work flawlessly in a virtual machines without hardware acceleration..

I'm also not able to use "swaymsg" commands in steam launch options... xrandr commands work though..

I do hope Wayland gets better but it has felt like a window manager that has huge limitations you'd expect to be resolved by now..I'm talking years of a feature not being present on Wayland like digital vibrance that you'd expect to be able to do... It not like it's impossible to configure or find solutions but sometimes there just isn't a solution when you use Wayland/Sway.. i3 is just easier hands down.

I do however like waybar.. i3status isn't as customizable (without extensive research) but it works very well either way.

**Comment by okraits** [comment]
I recommend i3blocks as a replacement for i3status. It has limitations as well but it's more customizable than i3status.
